Vagrantfile

```diff
@@ -81,7 +81,7 @@ Vagrant.configure("2") do |config|
      ./gtminstall --utf8 default
      gtmver=$(ls -1 /usr/lib/fis-gtm)
      echo "source /usr/lib/fis-gtm/$gtmver/gtmprofile" >> /home/ubuntu/.profile
-     ln -s $gtm_dist/libgtmshr.so /usr/local/lib
+     ln -s $gtm_dist/libgtmshr.so /vagrant/lib
```

Comment:
You don't want to do this. this has to be in /usr/local/lib for the linker to find it (on ubuntu)